ipfx/bin/run_feature_extraction.py

```diff
@@ -65,24 +65,16 @@ def run_feature_extraction(input_nwb_file,
                                ontology=ont,
                                api_sweeps=False)
 
-    try:
-        cell_features, sweep_features, cell_record, sweep_records = dsft.extract_data_set_features(data_set)
-
-        if cell_info: cell_record.update(cell_info)
-
-        cell_state = {"failed_fx": False, "fail_fx_message": None}
+    cell_features, sweep_features, cell_record, sweep_records, cell_state = dsft.extract_data_set_features(data_set)
 
-        feature_data = { 'cell_features': cell_features,
-                         'sweep_features': sweep_features,
-                         'cell_record': cell_record,
-                         'sweep_records': sweep_records,
-                         'cell_state': cell_state
-                         }
+    if cell_info: cell_record.update(cell_info)
```

Comment:
I know this was a single line in the original, but could you put the `cell_record.update(cell_info)` on the next line?